src/uu/du/src/du.rs

```diff
@@ -257,6 +263,18 @@ fn read_block_size(s: Option<&str>) -> usize {
     }
 }
 
+fn choose_size(matches: &ArgMatches, stat: &Stat) -> u64 {
+    if matches.is_present(options::INODES) {
+        stat.inodes
+    } else if matches.is_present(options::APPARENT_SIZE) || matches.is_present(options::BYTES) {
+        stat.size
```

Comment:
could you please add a test to cover this case? thanks

Reply:
The `--apparent-size`, right? Sure.

Reply:
please :)

Reply:
Updated